### PR TITLE
feat(react): change experimental Delay, SuspensiveProvider to stable

### LINKED ISSUE #30: [Feature]: new public api documentation
### **Package Scope**

- [x] Add to an existing package

Package name: @suspensive/react

### **Overview**

- [ ] Delay
- [ ] SuspensiveProvider, SuspensiveConfigs
- [ ] suspenseCache, useSuspenseCache

### LINKED ISSUE #111: [Feature]: Move experimental features to stable
### **Package Scope**

- [x] Add to an existing package
- [ ] New package

Package name: @suspensive/react

### PR DESCRIPTION
**Pull request by manudeli**
fix #111 fix #30

# Delay, SuspensiveProvider experimental to stable

Experimental to Stable
1. Delay
2. SuspensiveProvider

## PR Checklist

- [x] I have written documents and tests, if needed.

### CODE REVIEW

**Review by manudeli — COMMENTED** [review]
I left my intentions

File: packages/react/src/SuspensiveProvider.ko.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      ms: 1200,
+    },
+    suspense: {
+      fallback: <Spinner />,
+    },
+  },
+})
+
+function App() {
+  return <SuspensiveProvider value={suspensive}>...</SuspensiveProvider>
```
Comment (okinawaa) [inline packages/react/src/SuspensiveProvider.ko.mdx:23]: I have a question.!

If we are using both the `Delay` component and `Suspense` at the same time within the SuspensiveProvider,
like below code
```tsx

<SuspensiveProvider value={ms:2000}> // pseudo code
  <Suspense>
    const {data} =  useSuspenseQuery()
  </Suspense>
</SuspensiveProvider>

```

It takes 2000ms? or 2000ms + api call(network)


Does Suspense's asynchronous call only occur when Delay ends?

Comment (manudeli) [reply packages/react/src/SuspensiveProvider.ko.mdx:23]:
```tsx
const suspensive = new Suspensive({
  defaultOptions: {
    delay: {
      ms: 2000
    },
    suspense: {
      fallback: <Spinner /> 
    }
  }
})

<SuspensiveProvider value={suspensive}> // pseudo code
  <Delay>  // not showing during 2000ms and then below component will be showen
    <Suspense>  // after delayed by parent Delay, api will be called. then during api pending, default fallback Spinner(by Suspensive's defaultOptions) will show
      const {data} =  useSuspenseQuery()
    </Suspense>
  </Delay>
</SuspensiveProvider>
```

but I want to make users use like below.

```tsx
const suspensive = new Suspensive({
  defaultOptions: {
    delay: {
      ms: 2000
    }
  }
})

<SuspensiveProvider value={suspensive}> // pseudo code
  <Suspense fallback={<Delay><Spinner /></Delay>}>
    const {data} =  useSuspenseQuery()
  </Suspense>
</SuspensiveProvider>
```

Comment (manudeli) [reply packages/react/src/SuspensiveProvider.ko.mdx:23]: After I coding, I felt defaultOptions is not options. just `defaultProps`?

Comment (okinawaa) [reply packages/react/src/SuspensiveProvider.ko.mdx:23]: Thank you for the detailed explanation.!

I think option naming is good as it is(`defaultOptions`)
They don't go into the components, they go into the class called new Suspensive

**Review by okinawaa — APPROVED** [review]

**Pull request merged**